autogpt_platform/backend/backend/util/metrics.py

```diff
@@ -25,6 +26,7 @@ def sentry_init():
         environment=f"app:{settings.config.app_env.value}-behave:{settings.config.behave_as.value}",
         _experiments={"enable_logs": True},
         integrations=[
+            AsyncioIntegration(),
```

Comment:
How does this help us?

I have no idea what they do, and we need to have a common understanding so we dont replicate effort